src/databricks/labs/ucx/source_code/dbfsqueries.py

```diff
@@ -0,0 +1,38 @@
+from collections.abc import Iterable
```

Comment:
rename file to `dbfs.py`

Reply:
Semantically it would make sense

Reply:
We used the file name dbfs.py when resolving issue #1104. Would you like me to add the SQL linter into this file, or would you prefer a different name?

Reply:
OK. I will do that.